readme.md

```diff
@@ -143,6 +143,13 @@ You can optionally pass one or more of these parameters every time you use Lambo
   lambo new superApplication --path=~/Sites
   ```
 
+- `-f` or `--force` to force install even if the directory already exists 
+
+  ```bash
+  # Creates a new Laravel application after deleting ~/Sites/superApplication  
+  lambo new superApplication --force =~/Sites
```

Comment:
Do you have to pass a parameter to force? if so, can we get around that? Either way, why is there a space before the `=`?